pkg/parser/buildah/parser_test.go

```diff
@@ -318,7 +318,7 @@ func TestParser_Resolve(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			p := &Parser{}
-			got, err := p.Resolve(tt.args.fileContent, tt.args.filename)
+			got, err := p.Resolve(tt.args.fileContent, tt.args.filename, true)
```

Comment:
since this is a test, the value of the resolvereferences should take the default value of false no?